### ISSUE TITLE
Why website isnt up to date like Readme? 

### ISSUE THREAD

**Issue by inson1** [hackerkid/Mind-Expanding-Books#476]
Readme
![image](https://github.com/hackerkid/Mind-Expanding-Books/assets/75314629/0c257454-283b-4d6e-b253-7f659b1cf3e9)
Website
![image](https://github.com/hackerkid/Mind-Expanding-Books/assets/75314629/a8d90802-e2f5-4492-9bc6-09d627f8fafe)

**Comment by Yubraj977** [comment]
Yep that's the problem i see .  Admin please explain something and assign it to me i would love to work in it
